documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -2175,6 +2176,67 @@ If a temporal type uses the `SYSDATE` constant to represent the default value, t
 
+[[debezium-oracle-connector-converters]]
+== Custom converters
+
+The {prodname} Oracle connector provides several `CustomConverter` implementations specific for Oracle out of the box.
+These custom converters provide alternative mappings for specific data types influenced by configuration.
+To add a `CustomConverter` to the connector, follow the instructions in the link:../development/converters.adoc[Custom Converters documentation].
+
+=== `NUMBER(1)` to Boolean
+
+It wasn't until Oracle 23 that the Oracle database had a `BOOLEAN` logical data type.
+In earlier versions of Oracle, this was simulated using a `NUMBER(1)` data type, constrained with a value of `0` for false and a value of `1` for true.
+
+By default, such columns are emitted as an `INT8` literal type in change events.
+This may not be ideal in all situations and for use cases where it may be helpful to emit such columns as a logical `BOOL` type instead, you can use the `NumberOneToBooleanConverter` to achieve that outcome.
+
+.Example configuration
+[source]
+----
+converters=number-to-boolean
+converters.number-to-boolean.type=io.debezium.connector.oracle.converters.NumberOneToBooleanConverter
+converters.number-to-boolean.selector=.*.MY_TABLE.DATA
+----
+
+In this example, the `selector` property is optional, which allows specifying a regular expression to designate which tables or columns will be converted.
+If the `selector` property is omitted, the converter will convert all `NUMBER(1)` column types to logical `BOOL` field types in change events.
+
+=== `NUMBER` To Zero Scale
+
+Oracle allows for creating `NUMBER` based columns with negative scale, i.e. `NUMBER(-2)`.
+This can cause issues when being converted to Avro or when being handled by other systems that may not support negative scale values.
+
+.Example configuration
+[source]
+----
+converters=number-zero-scale
+converters.number-zero-scale.type=io.debezium.connector.oracle.converters.NumberToZeroScaleConverter
+converters.number-zero-scale.decimal.mode=precise
+----
+
+In this example, the `decimal.mode` property is optional, allowing to configure how decimal values are emitted.
+If the `decimal.mode` property is omitted, the converter will default to using `PRECISE` decimal handling mode.
+
+=== `RAW` to String
+
+While Oracle recommends not using certain data types such as `RAW`, legacy systems may continue to make use of this column type.
+By default, `RAW` column types are emitted as logical `BYTES`, as the column allows for storing binary or text-based data.
+
+In some cases, it may be desired for where `RAW` columns store character data as a series of bytes, to emit that column as `STRING` data for easier consumption by consumers.
+The `RawToStringConverter` provides a way to easily target such columns and to have the values emitted as strings rather than bytes.
```

Comment:
```suggestion
In some cases, `RAW` columns might store character data as a series of bytes.
To facilitate consumption by consumers, you can configure {prodname} to use the `RawToStringConverter`.
The `RawToStringConverter` provides a way to easily target such `RAW` columns and emit values as strings, rather than bytes.
The following example shows how to add the `RawToStringConverter` to the connector configuration:
```